src/App.css

```diff
@@ -0,0 +1,3 @@
+.App {
```

Comment:
make this sass file, then check that everything still loads through webpack. You might need sass-loader

Reply:
It still seems to work even without ejecting webpack... maybe webpack already accounts for sass-loader and we can actually avoid ejecting webpack altogether.